sklearn/feature_extraction/tests/test_dict_vectorizer.py

```diff
@@ -77,6 +77,23 @@ def test_one_of_k():
     assert_false("version" in names)
 
 
+def test_iterable_value():
+    D_in = [{"version": ["1", "2"], "ham": 2},
```

Comment:
Please also test `fit_transform` and `transform` where this is a finite generator rather than a list